sycl/source/detail/event_impl.hpp

```diff
@@ -234,6 +234,13 @@ class event_impl {
   /// state.
   bool isInitialized() const noexcept { return MIsInitialized; }
 
+  // Sets a sync point which is used when this event represents an enqueue to a
+  // pi_ext_command_buffer.
+  void setSyncPoint(pi_ext_sync_point SyncPoint) { MSyncPoint = SyncPoint; }
+
+  // Get the sync point associated with this event.
+  pi_ext_sync_point getSyncPoint() const { return MSyncPoint; }
```

Comment:
This file uses `RT::PiEvent` for a `pi_event`, wonder if we should define an alias here https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/detail/pi.hpp#L146 and use `RT::PiExtSyncPoint`.

Maybe add an alias for `pi_ext_command_buffer` too while we're at it.

Reply:
I've add aliases for the relevant command buffer classes and updated the usage inside our graphs implementation as well!